example/crd/federatedbar_test.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: federation.example.io/v1alpha1
+kind: FederatedBar
+metadata:
+  name: test-crd
+  namespace: test-namespace
+spec:
+  template:
+    metadata:
+      labels:
+        app: fed-crd
+    spec:
+      data: "Hello"
```

Comment:
I'd like you split this file to three files

```
federatedbar-template.yaml
federatedbar-placement.yaml
federatedbar-override.yaml
```

If you put this in one file, then we will always see the data is `World`, I'd like you apply `federatedbar-override.yaml` after finished the test of `federatedbar-template.yaml` and `federatedbar-placement.yaml`.

Reply:
In my tests, I saw `Hello` in one cluster and `World` in another.
One file is easy to observe, so I'd prefer to leave it as a single file.

Reply:
I'm also in favor of leaving things in a single file - it's much easier to share as an example that way.